rbd/rbd_pacific.go

```diff
@@ -0,0 +1,79 @@
+// +build !octopus,!nautilus
+
+package rbd
+
+// #cgo LDFLAGS: -lrbd
+// /* force XSI-complaint strerror_r() */
+// #define _POSIX_C_SOURCE 200112L
+// #undef _GNU_SOURCE
+// #include <errno.h>
+// #include <stdlib.h>
+// #include <rados/librados.h>
+// #include <rbd/librbd.h>
+import "C"
+
+import (
+	"unsafe"
+)
+
+const (
+	RBD_ENCRYPTION_FORMAT_LUKS1 = 0
+	RBD_ENCRYPTION_FORMAT_LUKS2 = 1
+)
```

Comment:
Instead of re-defining these constants, you can make them from a dedicated type and take the values from the C header:
```go
type EncryptionFormat C.uint

const (
    EncryptionFormatLUKS1 = EncryptionFormat(C.RBD_ENCRYPTION_FORMAT_LUKS1)
    EncryptionFormatLUKS2 = EncryptionFormat(C.RBD_ENCRYPTION_FORMAT_LUKS2)
)
```

This can then be passed to `EncryptionFormat()` as a more telling type than `format uint`.

The same can be done for the encryption algorithm, of course.